src/components/List.js

```diff
@@ -0,0 +1,77 @@
+import React, { Component } from 'react'
+import styled from 'styled-components'
+import Colors from './../constants/Colors'
+import AuthRequired from './AuthRequired'
+
+class Status extends Component {
+    constructor (props) {
```

Comment:
constructor is not needed there, we do not leverage any of its functionalities.
`people` could be moved outside of the component as well, as it is static now.

As the `Status` does not use any lifecycle methods or state itself, it could be fine to implement it as FunctionalComponent

Reply:
People is an currently mocked array and will be removed when backend response will be applied :)
FunctionalComponent is a good idea here, thanks 😊